docs/nodes/widgets/ui-chart.md

```diff
@@ -96,8 +100,13 @@ The next most important properties to configure are the "Chart Type" and "X-Axis
 - **Chart Type**: Choose between a Line, Scatter, or Bar chart.
 - **X-Axis Type**: Choose between a "Timescale" (for time-based data), "Linear" (for numerical data), or "Categorical" (for non-numeric data). You'll notice that some x-axis types are only available for certain chart types.
 
+
```

Comment:
<img width="805" alt="Screenshot 2024-07-26 at 11 14 14" src="https://github.com/user-attachments/assets/12373377-5858-41a2-9f4f-bd9502b80b6e">

Vertical alignment is off here

Reply:
Removed these additional "Try Demo" button as per discussion